src/MetaCompilation.Analyzers/Core/MetaCompilation.Analyzers.csproj

```diff
@@ -12,4 +12,5 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis" Version="$(MicrosoftCodeAnalysisVersion)" />
   </ItemGroup>
+  <Import Project="..\..\Utilities\Compiler\Analyzer.Utilities.projitems" Label="Shared" />
```

Comment:
`MetaCompilation.*.csproj` - these are not shipping projects. I wouldn't worry about optimizing these.

Reply:
Removed reference from project MetaCompilation.Analyzers to shared project Analyzer.Utilities.